src/pages/workspace/expensifyCard/WorkspaceExpensifyCardBankAccounts.tsx

```diff
@@ -0,0 +1,95 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React from 'react';
+import {View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import getBankIcon from '@components/Icon/BankIcons';
+import * as Expensicons from '@components/Icon/Expensicons';
+import MenuItem from '@components/MenuItem';
+import ScreenWrapper from '@components/ScreenWrapper';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@navigation/Navigation';
+import type {SettingsNavigatorParamList} from '@navigation/types';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import type SCREENS from '@src/SCREENS';
+import type {BankName} from '@src/types/onyx/Bank';
+import {isEmptyObject} from '@src/types/utils/EmptyObject';
+
+type WorkspaceExpensifyCardBankAccountsProps = StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.EXPENSIFY_CARD_BANK_ACCOUNT>;
+
+function WorkspaceExpensifyCardBankAccounts({route}: WorkspaceExpensifyCardBankAccountsProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const [bankAccountsList] = useOnyx(ONYXKEYS.BANK_ACCOUNT_LIST);
+
+    const policyID = route?.params?.policyID ?? '-1';
+
+    const handleAddBankAccount = () => {
+        // TODO: call to API - UpdateCardSettlementAccount
+        Navigation.navigate(ROUTES.BANK_ACCOUNT_WITH_STEP_TO_OPEN.getRoute('new', policyID, ROUTES.WORKSPACE_EXPENSIFY_CARD.getRoute(policyID)));
+    };
+
+    const handleSelectBankAccount = () => {
+        Navigation.navigate(ROUTES.WORKSPACE_EXPENSIFY_CARD_ISSUE_NEW.getRoute(policyID));
+    };
+
+    const renderBankOptions = () => {
+        if (!bankAccountsList || isEmptyObject(bankAccountsList)) {
+            return null;
+        }
+
+        // const eligibleBankAccounts = Object.values(bankAccountsList).filter((bankAccount) => bankAccount.accountData.allowDebit || bankAccount.accountData.type === CONST.BANK_ACCOUNT.TYPE.BUSINESS);
+        const eligibleBankAccounts = Object.values(bankAccountsList);
```

Comment:
Why did we omit this check, we could have added it to the mock data if needed

Reply:
will be held in https://github.com/Expensify/App/pull/46564